### PR TITLE
Fix ChatMemberUpdated.difference

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
Fixes a bug that's due to `ChatMember` no longer having all the attributes of its new subclasses.

### Checklist for PRs

- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [x] Created new or adapted existing unit tests
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs

### CODE REVIEW

**Comment by harshil21** [comment]
perhaps we can also adjust its test by testing different `ChatMember` classes? For e.g. `ChatMemberMember` to `ChatMemberAdministrator`

**Comment by Bibo-Joshi** [comment]
yes, of course - left the checklist unfilld on purpose ;)

File: tests/test_chatmemberupdated.py [hunk 1]
```diff
@@ -239,3 +241,19 @@ def test_difference_optionals(self, optional_attribute, user, chat):
...
+            chat=chat,
+            from_user=user,
+            date=datetime.datetime.utcnow(),
+            old_chat_member=old_chat_member,
+            new_chat_member=new_chat_member,
+        )
+        diff = chat_member_updated.difference()
+        assert diff.pop('is_anonymous') == (False, None)
+        assert diff.pop('until_date') == (None, datetime.datetime(2021, 1, 1))
+        assert diff.pop('status') == (ChatMember.CREATOR, ChatMember.KICKED)
```
Comment (harshil21) [inline tests/test_chatmemberupdated.py:258]: the `ChatMember.KICKED` makes me think that we could rename that to `ChatMember.BANNED`? Maybe in #2817?

**Review by harshil21 — APPROVED** [review]
Looks good just one thought:

File: tests/test_chatmemberupdated.py (hunk 1 above)
Comment (Bibo-Joshi) [reply tests/test_chatmemberupdated.py:258]: Sounds good, will do :)

**Pull request merged**